SConstruct

```diff
@@ -878,6 +898,15 @@ logger.info(textwrap.indent(cantera_conf, "    "), print_level=False)
 # *** Configure system-specific properties ***
 # ********************************************
 
+loglevel = env["logging"]
+if loglevel != "default":
+    logger.logger.setLevel(loglevel.upper())
```

Comment:
I tried to keep SCons settings and logger settings separate as much as possible. Fwiw, `--silent` is a native SCons command, so it does act on much of the SCons output. I opted for tying reduced logging output to this as I felt this would make intuitive sense.
```
scons build --silent logging=info
```
will suppress SCons ouptut, but still create logging output (where the `logging=info` acts as an override). SCons also supports the `-Q` option (which is more aggressive than `--silent`), but I have not found a way to capture whether it is toggled within `SConstruct`.